src/main/default/lwc/caseAssistFlow/caseAssistFlow.js

```diff
@@ -103,6 +104,27 @@ export default class CaseAssistFlow extends LightningElement {
     return reasonSuggestions;
   }
 
+
+  /* 
+  This sample component lets you specify a field for classification. Copy it every time you want to add a new field for classification. Replace <FIELDLABEL> 
+  with the label you plan to give to the field that you are configuring and <COVEOFIELDLABEL> with the Coveo field label 
+  configured in your Case Assist configuration in the Coveo Cloud Platform. 
```

Comment:
@erocheleau  Just to confirm:
1. These names should match, right? If so, I would also add "Note that these names must match."
2. Out of curiosity, we recommend updating the cookbook files first & then updating the Case Classification config in the Admin Console. However, could we do it the other way around (i.e., update Case Classification config first)?

Reply:
1. I will let Etienne answer, since I am not sure. 
2. They can do it in any order. Where do we recommend do one of them first?

Reply:
The order is implied in the introduction (i.e.,) "...After modifying these files, access the Coveo Administration Console to update your Case Classification configuration accordingly. Make sure that all newly added fields are specified in your configuration. For more information, see https://docs.coveo.com/en/3328/service/manage-case-assist-configurations#configuring-the-case-classification-functionality. "

Reply:
The field names should match the name of the coveo field in the source mappings.

If they are not available in the case assist config, they just won't get predictions.